src/styles/price.css

```diff
@@ -0,0 +1,4 @@
+.price__tag {
+  font-size: 16px;
+  font-weight: 700;
+}
```

Comment:
if you do not have the block price, maybe you do not need this?

Reply:
Kinda the same problem as in the comment above, but still agree, deleted.